scripts/setup-env.sh

```diff
@@ -1,28 +0,0 @@
-#!/usr/bin/env bash
```

Comment:
I think we want a file that includes these actions. Ideally a file like that used by Anaconda (?) where one executes wget to fetch the file and passes it to bash.

Reply:
Yes, I think so. We still need to clean up the install process to make it simpler. I still want a docker image that would work well actually. But this is just  small step as I realized a) we don't have those env variables collected anywhere and b) that other script was outdated and should be removed.